### PR TITLE
change the order of softmax and expand_dims

### PR DESCRIPTION
**Pull request by kelvinleen**
see [https://github.com/minsangkim142/R-net/issues/8](url)
just change the order, haven't verify.  sorry for that.

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Please also switch the name of the input s so that ,
scores = softmax(p1_logits)
scores = expand_dims(scores,-1)

**Comment by kelvinleen** [comment]
appology for my stupid fault.
